src/test/java/abel/ArrayListTest.java

```diff
@@ -0,0 +1,119 @@
+package abel;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+
+/**
+* Class to test Linked List methods.
+*/
+public class ArrayListTest {
+
+    private List arrayList;
+
+    /**
+     *
+     */
+    @Before
+    public void setUp() {
+        arrayList = new ArrayList<Integer>();
+    }
+
+    /**
+     *
+     */
+    @Test
+    public void testArrayListIsEmptyAtInitialize() {
+        assertTrue(arrayList.isEmpty());
+    }
+
+    /**
+     *
+     */
+    @Test
+    public void testArrayListReturnsNullIfItsEmpty() {
+        assertNull(arrayList.get(0));
```

Comment:
I think you should throw an exception in case the client is requesting a position that is not in list, consider the case you have a list full of custom objects and one of them is null the null in this case is a valid value it could have some meaning for the user and if the user has its own meaning for the null value what happens if the position that does not exists lets say 1000 and you return null what should the client do?